src/app.py

```diff
@@ -1695,6 +1695,8 @@ def ubkg_download_file_list():
     if not request.args or request.args.get('umls-key') is None:
         bad_request_error("Must include parameter 'umls-key'")
     umls_key = request.args.get('umls-key')
+    if umls_key is None or not umls_key.strip():
+        bad_request_error("Must include parameter 'umls-key'")
```

Comment:
Can't use the same error message since the `umls-key` is included this time but with empty value.

Also do the same check for `/umls-auth` which is causing the initial login error.